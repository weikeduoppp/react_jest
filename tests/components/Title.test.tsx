import React from "react";
import { render } from "@testing-library/react";
import Title from "components/Title";

// 快照更新 jest --updateSnapshot

describe('Title', () => {

  it("可以正确渲染小字", () => {
    const { baseElement } = render(<Title type="small" title="打字" />);
    expect(baseElement).toMatchSnapshot();
  });
})