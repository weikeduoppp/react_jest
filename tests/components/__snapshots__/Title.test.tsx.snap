// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Title 可以正确渲染小字 1`] = `
<body>
  <div>
    <p
      style="font-size: 0.5em; color: green;"
    >
      打字
    </p>
  </div>
</body>
`;
